browser/src/canvas/sections/ScrollSection.ts

```diff
@@ -1148,6 +1148,9 @@ export class ScrollSection extends CanvasSectionObject {
 	public onMouseWheel (point: Array<number>, delta: Array<number>, e: MouseEvent): void {
 		if (e.ctrlKey) return;
 
+		this.map.fire('closepopups'); // close all popups when scrolling
+		this.map._docLayer._resetReferencesMarks();
```

Comment:
does that function exists aslo outside of calc?

Reply:
isn't that causing that even if autofill popup is not opened, but we did some selection eg. for doing formula - it will disappear when scrolling?

type: `=SUM(`, select some range, scroll

Reply:
I think we want to just close popups -> then inside autofill implementation, it should reset references if is closing

Reply:
I think this is done with the new update.
Thanks for pointing out the issue on formula popup -> fixed.